duckduckgo/privacy-dashboard
# Malicious website protection - iOS support

Asana: https://app.asana.com/0/72649045549333/1208758247571066/f
**Reviewer:** @shakyShane 



## Description:

Enables Phishing and Malware warning for iOS

## Steps to test this PR:

1. Visit the[ preview](https://deploy-preview-252--ddgdash.netlify.app/app-debug/html/iframe?state=malware&platforms=ios)
2. Confirm that both phishing and malware states show on iOS
